l2geth/internal/ethapi/api.go

```diff
@@ -931,7 +923,6 @@ func DoCall(ctx context.Context, b Backend, args CallArgs, blockNrOrHash rpc.Blo
 	// and apply the message.
 	gp := new(core.GasPool).AddGas(math.MaxUint64)
 	if vm.UsingOVM {
```

Comment:
I'd like to have every instance of `vm.UsingOVM` documented with comments in the code itself